apps/service_providers/migrations/0020_auto_20241111_1113.py

```
@@ -219,5 +13,6 @@ class Migration(migrations.Migration):
         ('pipelines', '0008_pipelinechatmessages_node_id_and_more'),
     ]
 
-    operations = [migrations.RunPython(_create_llm_provider_models, reverse_code=_delete_all_llm_provider_models)
+    operations = [
+        # operations removed
     ]
```

Ruff (0.14.13)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)